nipoppy/sample_run_nipoppy.py

```diff
@@ -27,29 +36,85 @@
 log_file = f"{log_dir}/nipoppy.log"
 
 session_id = args.session_id
+session = f"ses-{session_id}"
+
+log_level = args.log_level.upper()
+
 n_jobs = args.n_jobs
 
-logger = my_logger.get_logger(log_file)
+logger = my_logger.get_logger(log_file, level=log_level)
 
 logger.info("-"*75)
 logger.info(f"Starting nipoppy for {DATASET_ROOT} dataset...")
-logger.info(f"dataset session (i.e visit): {session_id}")
+logger.info("-"*75)
+
+logger.info(f"dataset session (i.e visit): {session}")
 logger.info(f"Running {n_jobs} jobs in parallel")
 
 workflows = global_configs["WORKFLOWS"]
-logger.info(f"Running {workflows} serially")
+logger.info(f"Running workflows: {workflows} serially")
 
 for wf in workflows:
     logger.info("-"*50)
     logger.info(f"Starting workflow: {wf}")
-    if wf == "dicom_org":        
-        run_dicom_org.run(global_configs, session_id, n_jobs=n_jobs)
+    logger.info("-"*50)
+
+    if wf == "generate_manifest":
+        logger.info(f"***All sessions are fetched while generating manifest***")
+        # dataset specific
+        # generate_manifest.run(global_configs, task="regenerate", dash_bagel=True, logger=logger)
+        check_dicom_status.run(global_config_file, regenerate=True, empty=False)
+
+    elif wf == "dicom_org":        
+        run_dicom_org.run(global_configs, session_id, n_jobs=n_jobs, logger=logger)
+
     elif wf == "bids_conv": 
-        run_bids_conv.run(global_configs, session_id, n_jobs=n_jobs)
+        run_bids_conv.run(global_configs, session_id, n_jobs=n_jobs, logger=logger)
+
+    elif wf == "mriqc":
+        # Supported modalities (i.e. suffixes) for MRIQC
+        modalities = ["T1w", "T2w"]
+        
+        # Run mriqc tracker to regenerate bagel
+        dash_schema_file = f"{DATASET_ROOT}/proc/bagel_schema.json" 
```

Comment:
Maybe add a comment here saying that the schema needs to be downloaded from the dashboard repo. Or: add the schema to this repo like we said in #97.